### PR TITLE
Add message history using the up and down arrows.

### PR DESCRIPTION
**Pull request by LTAcosta**

### CODE REVIEW

**Comment by dashed** [comment]
@LTAcosta i dont quite understand this feature...

**Comment by LTAcosta** [comment]
It's a common feature in terminals and irc clients. Basically, pressing the up arrow key lets you retrieve past messages you have sent, in case you need to send them again. This works across channels in case you need to send the same message to multiple channels.

**Comment by dashed** [comment]
@LTAcosta ah i see. Will you be able to rebase this?

**Comment by dashed** [comment]
@LTAcosta is this good for merge?

**Comment by LTAcosta** [comment]
Yep

File: robin.user.js [hunk 1]
```diff
@@ -846,25 +846,74 @@
...
+		pastMessageTemp = "";
+		var value = $("#robinMessageTextAlt").val();
+		
+		if (!value || (pastMessageQueue.length > 0 && value == pastMessageQueue[0]))
+			return;
+		
+		pastMessageQueue.unshift(value);
+		
+		// Currently only storing the past 50 messages. 
+		if (pastMessageQueue.length > 50)
```
Comment (dashed) [inline robin.user.js:865]: Make length a config setting.

**Pull request merged**